### DISCUSSION TITLE
Multiple SecuritySchemes

### DISCUSSION THREAD

**Discussion by adonai36** [dedoc/scramble#192] [Q&A]
Is it possible to create multiple SecuritySchemes to be adhered to?

**Comment by Wulfheart** [comment]
I guess it is possible at least with an extension.

**Comment by sprklinginfo** [comment]
I am having the same issue, I added  SecurityScheme::http('basic') in the service provider according to the docs. but this is a legacy project, older APIs use basic, but newer APIs use bearer. Is it possible manually to add `bearer`, kind of 'override' the global setting, on relevant APIs? @adonai36, have you found an answer?
